pkg/k8s/service.go

```diff
@@ -493,15 +522,19 @@ func NewClusterService(id ServiceID, k8sService *Service, k8sEndpoints *Endpoint
 // ParseClusterService() is paired with EqualsClusterService() that
 // has the above wired in.
 func ParseClusterService(svc *serviceStore.ClusterService) *Service {
-	var ip net.IP
+	feIPs := make([]net.IP, len(svc.Frontends))
 	var ipStr string
 	ports := serviceStore.PortConfiguration{}
-	for ipStr, ports = range svc.Frontends {
-		ip = net.ParseIP(ipStr)
-		break
+
+	i := 0
+	for ipStr = range svc.Frontends {
+		feIPs[i] = net.ParseIP(ipStr)
+		i++
```

Comment:
```
for i, ip := range svc.Frontends {
 	feIPs[i] = net.ParseIP(ipStr) 
}
```

Reply:
@christarazi `svc.Frontends` is a map, so `i, ip` will return the "key" and the "value.

Reply:
Thanks, clearly I wasn't paying too close of attention :facepalm: